src/DnsHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Yiisoft\NetworkUtilities;
+
+class DnsHelper
+{
+    /**
+     * @param string $hostname hostname without dot at end
+     */
+    public static function checkMx(string $hostname): bool
+    {
+        $hostname .= '.';
+        try {
+            if (!@dns_check_record($hostname, 'MX')) {
+                return false;
+            }
+            $result = @dns_get_record($hostname, DNS_MX);
+            return $result !== false && count($result) > 0;
+        } catch (\Throwable $t) {
+        }
+        return false;
+    }
+
+    public static function checkA(string $hostname): bool
```

Comment:
```suggestion
    public static function checkAExists(string $hostname): bool
```

Reply:
How about A and C records then? I think naming of these IP would add to confusion.